### ISSUE TITLE
Did you try to use random aspect-ratio of mask, limit lambda values and insert image not at the edge?

### ISSUE THREAD

**Issue by AlexeyAB** [clovaai/CutMix-PyTorch#14]
@hellbell Hi,

1. Did you try to use random aspect-ratio of mask (in some range) rather than proportional to the original image?

2. Have you tried to limit the possible values of lambda in the range of, for example: 0.5 - 0.7 (instead of 0.0 - 1.0)?

![image](https://user-images.githubusercontent.com/4096485/69901766-07331880-1397-11ea-96a0-e6103f11fb1b.png)

----

3. In all your examples, the inserted image is always adjacent to the edge of the original image. Does your code only use this approach? And why?

![image](https://user-images.githubusercontent.com/4096485/69901857-22eaee80-1398-11ea-872a-410f8e11968e.png)

![image](https://user-images.githubusercontent.com/4096485/69901865-3433fb00-1398-11ea-934a-2484b59613b3.png)

**Comment by hellbell** [comment]
@AlexeyAB 

1 and 2:
No, I didn't try that. But when applying that kind of factor, we can expect to obtain better performance with a proper hyperparameter tuning. 

3:
No, our code does not sample the cropping region with any boundary-aware constraint. 

Thanks!
